colorschemes/dark-abc.conf

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
Would you mind adding a full file header with license and copyright information?
Like this one: https://github.com/geany/geany-themes/blob/master/colorschemes/github.conf